srcpkgs/vosk-api/template

```diff
@@ -0,0 +1,79 @@
+# Template file for 'vosk-api'
+pkgname=vosk-api
+version=0.3.43
+revision=1
+create_wrksrc=yes
+build_wrksrc="vosk-api-${version}/src"
+_openfst_commit=7dfd808194105162f20084bb4d8e4ee4b65266d5
```

Comment:
I don't see this used anywhere, is this here from when openfst was vendored in this package?